src/Service/DeliverySettingsProvider.php

```diff
@@ -250,6 +250,17 @@ private function getPrice(int $carrierId, string $name, float $taxRate = 1): flo
         return Tools::ps_round($price * $taxRate, 2);
     }
 
+    /**
+     * @param  string $setting
+     *
+     * @return null|string
+     */
+    private function getString(string $setting): ?string
```

Comment:
`getString` vind ik persoonlijk niet de juiste benaming. Pas als je in de method kijk zie je dat het voor het ophalen van een configuratie is.